src/Arcus.BackgroundJobs.AzureActiveDirectory/ClientSecretExpirationJobOptions.cs

```diff
@@ -0,0 +1,109 @@
+﻿using System;
+using System.Net.Mime;
+using CloudNative.CloudEvents;
+using GuardNet;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Arcus.BackgroundJobs.AzureActiveDirectory
+{
+    /// <summary>
+    /// Represents the additional options that the user can configure during the <see cref="IServiceCollectionExtensions.AddClientSecretExpirationJob"/> call.
+    /// </summary>
+    public class ClientSecretExpirationJobOptions
+    {
+        private int _runAtHour = 0;
+        private bool _runImmediately = false;
+        private Uri _eventUri = null;
+        private int _expirationThreshold = 14;
+
+        /// <summary>
+        /// Gets or sets the hour which to query for client secrets.
+        /// </summary>
+        /// <exception cref="ArgumentOutOfRangeException">Thrown when the <paramref name="value"/> is less than zero or greater than 23.</exception>
+        public int RunAtHour
+        {
+            get => _runAtHour;
+            set
+            {
+                Guard.NotLessThan(value, 0, nameof(value));
+                Guard.NotGreaterThan(value, 23, nameof(value));
+
+                _runAtHour = value;
+            }
+        }
```

Comment:
Would love to see maybe a some unit tests to verify these guards.
Don't know if I said it already, but great job abstracting the obscure CRON schedule to a user-friendly option 👍 .  Amazingly done.

Reply:
Thanks! 
Added some tests for the guards: [`2087f72` (#87)](https://github.com/arcus-azure/arcus.backgroundjobs/pull/87/commits/2087f728295ca6969978bcf35a9e23eb4bdcbdaa)